contracts/tokenomics/generator/src/contract.rs

```diff
@@ -1781,6 +1858,42 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, C
                     };
                     POOL_INFO.save(deps.storage, &Addr::unchecked(key), &pool_info)?;
                 }
+
+                migration::migrate_configs_to_v120(&mut deps, active_pools, msg)?
+            }
+            "1.1.0" => {
+                let msg: migration::MigrationMsgV110 = from_binary(&msg.params)?;
```

Comment:
At this version we have already set factory and generator controller addresses and only need setting the active pools

Reply:
It's necessary because the config also needs to be migrated after the last config update.